### PR TITLE
HRIN-412: pretix order lists

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/HRIN-412 (+ https://jira.itkdev.dk/browse/HRIN-447)

### CODE REVIEW

File: templates/itk-pretix-orders-date.html.twig [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <tbody>
+    {% for order in orders %}
+      {%  set order_url = order.getUrl %}
+    {% for position in order.getPositions %}
+      <tr>
+        <td><a href="{{  order_url }} ">{{ position.getOrder }}/{{ position.getSubevent }}</a></td>
+        <td>{{ position.getAttendeeName|default('–') }}</td>
+        <td>{{ position.getAttendeeEmail|default('–') }}</td>
+        <td>{{ position.getPrice }}</td>
+        {#<td>{{ dump(position) }}</td>#}
```
Comment (martinyde) [inline templates/itk-pretix-orders-date.html.twig]: In or out?

**Review by martinyde — CHANGES_REQUESTED** [review]
A single comment to a template

File: templates/itk-pretix-orders-date.html.twig (hunk 1 above)
Comment (rimi-itk) [reply templates/itk-pretix-orders-date.html.twig]: Out. Will fix.

**Comment by rimi-itk** [comment]
> A single comment to a template

I don't understand this comment …

**Pull request merged**